CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+
+
+## Features
```

Comment:
As per videocall, this changelog is generated automatically, no need to add it manually. Just put it in the git commit.